### PR TITLE
Fix for #326

### PR DESCRIPTION
**Pull request by canarslan12**
I hope that you didn't change this for some reason
https://github.com/igrigorik/videospeed/commit/e50c29c5dd521cb0b91ffcc522bfa3e54e002070

### CODE REVIEW

File: inject.js [hunk 1]
```diff
@@ -191,7 +191,7 @@ chrome.runtime.sendMessage({}, function(response) {
     window.onload = () => {
       initializeNow(window.document)
     };
-    if (document && document.doctype && document.doctype.name == "html") {
```
Comment (igrigorik) [inline inject.js]: Why is this being removed? This was added to address https://github.com/igrigorik/videospeed/issues/309

Comment (mariusi7) [reply inject.js]: I think that test makes the initialization not happen when the URL is smth like file:///C:/Users/myvideo.mp4 (?)

Comment (mariusi7) [reply inject.js]: https://i.imgur.com/DeJZjiN.png

Comment (igrigorik) [reply inject.js]: Ah, interesting.. doh. Yeah, let's find a different solution for #309 then. Could you take a look at the page flagged in #309 and see if we can find a different solution for it?

Comment (canarslan12) [reply inject.js]: I think my solution is not breaking it but might break it. why is it here?
https://github.com/igrigorik/videospeed/commit/7606224ec92790f65f27b348c9fb7346ecbc7d3e

**Comment by canarslan12** [comment]
I think it's good now

**Comment by mariusi7** [comment]
@igrigorik Please check and approve this merge. 
When can we expect a new final version that contains the fix for #326 ?

**Comment by igrigorik** [comment]
Sorry about the delay. Testing on this end.. seems to be working as promised!

**Comment by mariusi7** [comment]
Any plans for a new version 0.5.2 with this fix contained ? I'm using the development mode which always nags me on open

**Comment by igrigorik** [comment]
@mariusi7 yes, planning to release an update soon. Need to track down a few (unrelated, I believe) issues first.

**Pull request merged**